Sources/Queues/Queue.swift

```diff
@@ -82,7 +82,7 @@ extension Queue {
         return self.set(id, to: storage).flatMap {
             self.push(id)
         }.map { _ in
-            self.logger.info("Dispatched queue job", metadata: [
+            self.logger.info("Dispatched queue job: \(job)", metadata: [
```

Comment:
What is `job` here? If it's entire `JobData` object I don't think this should be printed in the logs. Regardless I think this should go into `metadata` not the actual log string

Reply:
yup, meant to be the name, I was a bit too quick there...